modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -197,6 +203,26 @@ public function renewLoans($ids) {
   }
 
   /**
+   * Create a reservation for a patron.
+   *
+   * @param AlephPatron $patron
+   * @param AlephReservation $reservation
+   *
+   * @throws \RuntimeException
+   *
+   * @return \Drupal\aleph\Aleph\Entity\AlephRequestResponse
+   */
+  public function createReservation($patron, $reservation) {
+    $response = $this->client->createReservation($patron,
+      $reservation->getRequest());
+
+    $request_response = new AlephRequestResponse();
+    $request_response::createRequestResponseFromXML($response);
+
+    return $request_response;
```

Comment:
This does not make sense to me. `createRequestResponseFromXML` is a static method with a return value. This tries to call it from an object instance but do not assign the return value.

This should be `return AlephRequestResponse::createRequestResponseFromXML($response);`